api.ts

```diff
@@ -12,45 +12,51 @@ interface Paginated<T> {
   items: T[];
 }
 
-interface Asset {
+export interface Asset {
   account_id: string;
   definition_id: string;
   value: AssetValue;
 }
 
-type AssetValue =
+export type AssetValue =
   | Tagged<"Quantity", number>
   | Tagged<"BigQuantity", bigint> // be careful! should be deserialized with `json-bigint`
   | Tagged<"Fixed", string> // it's a number too, "float" number, but it cannot fit into js `number`
   | Tagged<"Store", any>;
 
-type AssetValueType = "Quantity" | "BigQuantity" | "Fixed" | "Store";
+export type AssetValueType = "Quantity" | "BigQuantity" | "Fixed" | "Store";
 
-
-interface AssetDefinition {
+export interface AssetDefinition {
   id: string;
   value_type: AssetValueType;
   mintable: Mintable;
 }
 
-type Mintable = 'Once' | 'Infinitely' | 'Not'
+export interface AssetDefinitionWithAccounts extends AssetDefinition {
+  /**
+   * List of account IDs
```

Comment:
Be advised, this will change to an ss58-like scheme. Shortly it will look like a public key.